torch_ort_inference/docker/Dockerfile.ort-infer-stable-torch1120-onnxruntime1120-openvino-ubuntu20.04

```diff
@@ -0,0 +1,86 @@
+#-------------------------------------------------------------------------
+# Copyright(C) 2022 Intel Corporation.
+# SPDX-License-Identifier: MIT
+#--------------------------------------------------------------------------
+
+ARG OPENVINO_VERSION="2022.1.0"
+ARG OVPT_BRANCH="releases/v0.5"
+
+FROM openvino/ubuntu20_runtime:${OPENVINO_VERSION}
+
+LABEL description="This is the runtime image for OpenVINO™ integration with PyTorch on Ubuntu 20.04 LTS"
+
+LABEL vendor="Intel Corporation"
+
+USER root
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
+
+RUN apt-get update; \
+    apt-get install -y --no-install-recommends \
+    git wget build-essential; \
+    rm -rf /var/lib/apt/lists/*;
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1; \
+    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2;
+
+WORKDIR /home/ovtort/
+
+RUN git clone https://github.com/intel-staging/ort.git && \
+    cd ort/torch_ort_inference && \
+    git checkout ${OVPT_BRANCH} && \
+    git submodule init && \
```

Comment:
Git submodule and recursive steps not needed.

Reply:
fixed, please review for any discrepancies.